UnitsNet/QuantityFormatter.cs

```diff
@@ -142,9 +156,6 @@ private static string FormatUntrimmed<TUnitType>(IQuantity<TUnitType> quantity,
 
                 switch(formatSpecifier)
                 {
-                    case 'G':
-                    case 'g':
-                        return ToStringWithSignificantDigitsAfterRadix(quantity, formatProvider, 2);
```

Comment:
I think this change is for the better. It reduces the wtf-factor a bit by being more consistent with .NET, such as TimeSpan and numbers.  It also fixes the issue with `"g1"` not rounding to 1 decimals.

Precision errors will become more evident too when doing arithmetic, for people running into equality issues beyond units.

`ToString("s")` would equal today's behavior, and is hopefully fairly straight forward to setup in most UI frameworks.

However, I suspect a lot of people are using `ToString()` directly or indirectly in a lot of UI places so I think I would move it to v6.

Create a `release/v6` branch and target this PR there.